File: python/1234.py
while True:
    try:
        x = input()
        y = ''
        
        flag = True
        
        for i in range(len(x)):
            if x[i] != ' ':
                if flag == True:
                    y += x[i].upper()
                    flag = False
                elif flag == False:
                    y += x[i].lower()
                    flag = True
            else:
                y += ' '
        
        print(y)                    
        
    except:
        break

File: lua/1234.lua
local function main()
    local x = io.read()
    local newX = ''
    local capital = true
    for i = 1, #x do
        local char = string.sub(x, i, i)
        if char ~= ' ' then
            if capital then
                char = string.upper(char)
            else
                char = string.lower(char)
            end
            capital = not capital
        end
        newX = newX..char
    end
    print(newX)
end

while true do
    if not pcall(main) then
        break
    end
end